src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+  unique.
+  In the table below, time zones in the Conflicting Zones column are not available
+  through use of a timezone abbreviation.
+</Callout>
+
+It is possible to use timezone abbreviations in `SINCE`/`UNTIL` DateTime strings.
+
+<Collapser
+  className="freq-link"
+  title={<>Timezone Abbreviations</>}
+>
+  <table style={{width: "auto"}}>
+    <thead>
+    <tr>
+      <th>Abbreviation</th>
+      <th>Primary Zone</th>
+      <th>Other Supported Zones</th>
+      <th>Conflicting Zones</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>ACDT</td>
+      <td>Australian Central Daylight Time [+10:30]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ACST</td>
+      <td>Acre Standard Time [-05:00]</td>
+      <td></td>
+      <td>Australian Central Standard Time [+09:30]</td>
+    </tr>
+    <tr>
+      <td>ACT</td>
+      <td>Acre Standard Time [-05:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ACWST</td>
+      <td>Australian Central Western Standard Time [+08:45]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ADT</td>
+      <td>Atlantic Daylight Time [-03:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AEDT</td>
+      <td>Australian Eastern Daylight Time [+11:00]</td>
+      <td>Australian Eastern Daylight Time (Macquarie) [+11:00]</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AEST</td>
+      <td>Australian Eastern Standard Time [+10:00]</td>
+      <td>Macquarie Island Time [+10:00]</td>
+      <td>Australian Eastern Standard Time [+10:00]</td>
+    </tr>
+    <tr>
+      <td>AFT</td>
+      <td>Afghanistan Time [+04:30]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AKDT</td>
+      <td>Alaska Daylight Time [-08:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AKST</td>
+      <td>Alaska Standard Time [-09:00]</td>
+      <td>Gambier Time [-09:00]</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ALMT</td>
+      <td>East Kazakhstan Time [+06:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AMT</td>
+      <td>Amazon Standard Time [-04:00]</td>
+      <td></td>
+      <td>Armenia Standard Time [+04:00]</td>
+    </tr>
+    <tr>
+      <td>ANAT</td>
+      <td>Anadyr Standard Time [+12:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AQTT</td>
+      <td>West Kazakhstan Time [+05:00]</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ART</td>
+      <td>Argentina Standard Time [-03:00]</td>
+      <td>Western Argentina Standard Time [-03:00]</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>AST</td>
+      <td>Atlantic Standard Time [-04:00]</td>
+      <td></td>
+      <td>Arabian Standard Time [+03:00]</td>
+    </tr>
+    <tr>
+      <td>AWST</td>
+      <td>Australian Western Standard Time [+08:00]</td>
+      <td></td>
+      <td>Casey Time [+11:00]</td>
```

Comment:
The abbreviation for Casey Time appears to be CAST?

Reply:
So, the following code:
```java
        final TimeZone caseyTime = TimeZone.getTimeZone("Antarctica/Casey");
        System.out.println(caseyTime.getID());
        System.out.println(caseyTime.getDisplayName());
        System.out.println(caseyTime.getDisplayName(false, TimeZone.SHORT));
```
renders the following output:
```text
Antarctica/Casey
Casey Time
AWST
```
I'm not clear why Java is doing this, but that's why it's in the list that way.